app/models/API1811/FinancialTransactions.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.API1811
+
+import play.api.libs.json._
+
+case class FinancialTransactions(financialTransactions: Option[Seq[Transaction]] = None)
```

Comment:
This field name should be `financialDetails`; refer to the spec. I also don't think this type needs to be optional given it's a required field, and is of type array with a minimum number of 1 items.